cqrs-user-query-service/src/main/java/dev/amir/userquery/domain/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package dev.amir.userquery.domain.exception;
+
+public class UserNotFoundException extends UserException {
+    public UserNotFoundException(String userId) {
+        this("User with id:" + userId + " Not found",null);

```

Comment:
is there a `super(String)` constructor? you can call it instead of `this(String, Throwable)`